src/overlays/actors/ovl_En_Zo/z_en_zo.h

```diff
@@ -6,9 +6,38 @@
 
 struct EnZo;
 
+typedef struct {
+    /* 0x00 */ u8    type;
+    /* 0x04 */ f32   scale;
+    /* 0x08 */ f32   targetScale;
```

Comment:
lately we've started just putting one space after the type instead of trying to line things up. we dont have a set rule on this or anything, but just putting it out there. it would also help it match the style of the actor struct below :D